packages/suite/src/components/wallet/AccountsMenu/index.tsx

```diff
@@ -82,7 +84,7 @@ const Heading = styled(H2)<{ isMobileLayout?: boolean }>`
 const MenuItemsWrapper = styled.div`
     position: relative;
     width: 100%;
-    height: 100vh;
+    height: ${isDesktop() ? `calc(100vh - ${DESKTOP_TITLEBAR_HEIGHT})` : '100vh'};
```

Comment:
I am not sure about this one, just making note I'll test it later. but most likely it works correcly.
(this component is used only on mobile layout)

Reply:
I just wanted to keep it consistent but it might be better to remove it when user is not allowed to make the desktop window small enough to trigger this component. I'll do that 👍

Reply:
yeah that was more note to myself, I actually think there will be some visual bug as height 100vh for this component would mean it will get outside of viewport because there are some additional components on top that are eating from the 100vh 😁